plenum/test/client/test_core_authn.py

```diff
@@ -0,0 +1,151 @@
+import pytest
+
+from plenum.common.constants import GET_TXN, NODE, NYM
+from plenum.common.exceptions import CouldNotAuthenticate, \
+    InsufficientSignatures, InsufficientCorrectSignatures, MissingSignature
+from plenum.common.signer_simple import SimpleSigner
+from plenum.common.types import f
+from plenum.server.client_authn import CoreAuthNr
+
+
+idr = '5G72199XZB7wREviUbQma7'
+msg_str = "42 (forty-two) is the natural number that succeeds 41 and precedes 43."
+
+
+class DummyAuthenticator(CoreAuthNr):
+    def getVerkey(self, _):
+        return None
+
+
+@pytest.fixture(scope="module")
+def cli():
```

Comment:
Why is it called `cli`? It sound like command line interface.

Reply:
fixed

Reply:
Changed